### PR TITLE
Add vim modes subsection to commandline/vim.md doc 

### PR DESCRIPTION
**Pull request by hakanergul**
### Description

Vim modes (`command`, `insert` and `visual`) are added to `content/command-line/concepts/vim/vim.md` file. I just add a brief description for these modes. 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/command-line/concepts/vim/vim.md [hunk 1]
```diff
@@ -30,6 +30,13 @@ This section describes some important features of the Vim editor.
 - Users can create macros of a sequence of commands.
 - Users can save sessions to start where you left off last time.
 
+## Vim Modes
+Vim has three modes:
+
++ `COMMAND MODE` : *Execute commands like undo, redo, find, replace, quit, etc.*
++ `INSERT MODE` : *Type text*
++ `VISUAL MODE` : *Highlight, select and edit text*
```
Comment (SSwiniarski) [inline content/command-line/concepts/vim/vim.md:38]:
```suggestion
- `COMMAND MODE` : Execute commands like undo, redo, find, replace, quit, etc.
- `INSERT MODE` : Type and edit text.
- `VISUAL MODE` : Highlight, select and edit text.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@hakanergul I did a review and had one comment, mostly about formatting.

**Comment by hakanergul** [comment]
@SSwiniarski thank you. I committed the changes👍

**Review by SSwiniarski — APPROVED** [review]
@hakanergul I made one small change and it looks ready for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@hakanergul @SSwiniarski LGTM. Just made some minor edits, mostly on the existing text describing the features of vim.

**Comment by SSwiniarski** [comment]
@hakanergul congrats on the new edit,

Live link: https://www.codecademy.com/resources/docs/command-line/vim

**Pull request merged**